implants/lib/eldritchv2/stdlib/eldritch-libsys/src/std/exec_impl.rs

```diff
@@ -103,13 +130,20 @@ fn handle_exec(
                         exit(0);
                     }
                     ForkResult::Child => {
-                        let _res = Command::new(path)
+                        let mut res = Command::new(path)
```

Comment:
Double check this doesn't hold open a parent child relationship between imix and the process.